getting_started/5_Quantum_Teleportation.ipynb

```diff
@@ -0,0 +1,285 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Quantum Teleportation\n",
+    "\n",
+    "Quantum teleportation is a technique for moving quantum states around, even in the absence of a quantum communications channel linking the sender of the quantum state to the recipient.\n",
+    "\n",
+    "\n",
+    "Here’s how quantum teleportation works. Alice and Bob met long ago but now live far apart. While together they generated a Bell pair, each taking one qubit of the Bell pair when they separated. Many years later, Bob is in hiding, and Alice’s mission, should she choose to accept it, is to deliver a qubit $|ψ⟩$ to Bob. She does not know the state of the qubit, and moreover can only send classical information to Bob. Should Alice accept the mission?\n",
+    "\n",
+    "Alice doesn’t know the state $|ψ⟩$ of the qubit she has to send to Bob, and the laws of quantum mechanics prevent her from determining the state when she only has a single copy of $|ψ⟩$ in her possession. What’s worse, even if she did know the state $|ψ⟩$, describing it precisely takes an infinite amount of classical information since $|ψ⟩$ takes values in a continuous space. So even if she did know $|ψ⟩$, it would take forever for Alice to describe the state to Bob. It’s not looking good for Alice. Fortunately for Alice, quantum teleportation is a way of utilizing the entangled Bell pair in order to send $|ψ⟩$ to Bob, with only a small overhead of classical communication.\n",
```

Comment:
I find the point about "precisely describing the state takes infinite amount of classical information" a bit misleading.  Since real quantum computers have error, doing quantum teleportation introduces some error when the target state is teleported anyway. Describing the state up to some error certainly doesn't take an infinite amount of information.